google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/SequentialExecutorService.java

```diff
@@ -233,7 +245,11 @@ public void cancel(Throwable e) {
       return future;
     }
 
-    /** Cancels every task in the queue assoicated with {@code key}. */
+    void resumePublish(String key) {
```

Comment:
Is it intended that the function name in Publisher is resumePublishing and this one is resumePublish?

Reply:
I made both `resumePublish`